service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ReferralRequestMapper.java

```diff
@@ -0,0 +1,73 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import lombok.RequiredArgsConstructor;
+import org.hl7.fhir.dstu3.model.*;
+import org.springframework.stereotype.Component;
+import uk.nhs.adaptors.oneoneone.cda.report.service.HealthcareServiceService;
+
+import uk.nhs.connect.iucds.cda.ucr.*;
+
+import java.util.Date;
+
+import java.util.Base64;
+
+
+@Component
+@RequiredArgsConstructor
+public class ReferralRequestMapper {
+
+    private final HealthcareServiceService healthcareServiceService;
+    private final PatientMapper patientMapper;
+    private final EncounterMapper encounterMapper;
+
+    public ReferralRequest transform(POCDMT000002UK01ClinicalDocument1 clinicalDocument) {
+
+        //TODO: Pathways - to discuss with team
+
+//        String s = clinicalDocument.getComponent().getStructuredBody().getComponentArray(0)
+//                .getSection().getEntryArray(0).getObservationMedia().getValue().xmlText();
+//
+//        int startPosition = s.indexOf(">\n                ") + ">\n                ".length();
```

Comment:
I know this is commented out, but why this string of is so weird? What is the logic for this?

Reply:
This is what we need to discuss, see attachment in teams chat